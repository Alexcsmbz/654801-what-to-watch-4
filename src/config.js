export const moviePageTabs = [
  {
    name: `Overview`,
  },
  {
    name: `Details`,
  },
  {
    name: `Reviews`,
  },
];

export const RoutePath = {
  MAIN: `/`,
  LOGIN: `/login`,
  MOVIE: `/films/:id`,
  MYLIST: `/mylist`,
  REVIEW: `/films/:id/review`,
};
