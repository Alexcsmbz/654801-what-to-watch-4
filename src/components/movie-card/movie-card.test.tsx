import MovieCard from './movie-card';
import {BrowserRouter as Router} from 'react-router-dom';
import {testMovie} from 'config';

describe(`MovieCard snapshot test`, () => {
  const onClick = jest.fn(() => {});

  it(`MovieCard should render movie`, () => {
    const tree = renderer.create(
        <Router>
          <MovieCard
            movie={testMovie}
            onClick={onClick}
          />
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
