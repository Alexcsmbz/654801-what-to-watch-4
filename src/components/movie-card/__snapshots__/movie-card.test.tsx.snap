// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard snapshot test MovieCard should render movie 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
>
  <div
    onClick={[Function]}
  >
    <video
      controls={true}
      height="175"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      poster="previewImage"
      width="280"
    >
      <source
        src="previewVideoLink"
        type="video/mp4"
      />
    </video>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/1"
        onClick={[Function]}
      >
        name
      </a>
    </h3>
  </div>
</article>
`;
